GCRCatalogs/cosmodc2.py

```diff
@@ -600,6 +602,74 @@ def _generate_quantity_modifiers(self):
         return quantity_modifiers
 
 
+class DiffSkyGalaxyCatalog(CosmoDC2ParentClass):
+    """
+    DiffSky galaxy catalog reader, inherited from CosmoDC2ParentClass
+    Class for new generation of catalogs generated with JAX-based 
+    forward modeling techniques.
```

Comment:
Suggest adding what catalogs use this reader
```suggestion
    forward modeling techniques.
    This reader is used by SkySim v3+ (not SkySim5000) catalogs.
```